src/App.tsx

```diff
@@ -16,35 +19,69 @@
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const handleReset = () => {
+    setSortField(SortType.DEFAULT);
+    setIsReversed(false);
+  };
+
+  const sortedGoods = getSortedGoods(goodsFromServer, {
+    sortField,
+    isReversed,
+  });
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={cn('button is-info', {
+            'is-light': sortField !== SortType.ALPHABETIBALLY,
+          })}
+          onClick={() => setSortField(SortType.ALPHABETIBALLY)}
```

Comment:
[UX] There's no visual indication of the active sorting method. Users should be able to see which sorting method is currently active. Consider adding a class to highlight the active button.